### PR TITLE
fix(ibuffer): Wrong number of arguments to ibuffer-find-file with ivy enabled.

### PR DESCRIPTION
**Pull request by p4v4n**
Error message on C-x C-f in `ibuffer` :

```
(wrong-number-of-arguments ((t) (_file &optional _wildcards) "Use `counsel-find-file' instead of `find-file'." (interactive) (counsel-find-file (let ((buf (ibuffer-current-buffer))) (if (buffer-live-p buf) (save-current-buffer (set-buffer buf) default-directory) default-directory)))) 0)
```

### CODE REVIEW

File: modules/emacs/ibuffer/config.el [hunk 1]
```diff
@@ -57,11 +57,11 @@
       (+ibuffer-workspace (+workspace-current-name))))
 
   (when (featurep! :completion ivy)
-    (defadvice! +ibuffer-use-counsel-maybe-a (_file &optional _wildcards)
+    (defadvice! +ibuffer-use-counsel-maybe-a (&optional _file _wildcards)
```
Comment (hlissner) [inline modules/emacs/ibuffer/config.el:60]: Please revert this change. `ibuffer-find-file`'s first argument [is not optional](https://github.com/emacs-mirror/emacs/blob/master/lisp/ibuffer.el#L863).

**Review by hlissner — CHANGES_REQUESTED** [review]

File: modules/emacs/ibuffer/config.el (hunk 1 above)
Comment (p4v4n) [reply modules/emacs/ibuffer/config.el]: @hlissner Made the necessary changes to retain the function signature.

**Review by hlissner — APPROVED** [review]

**Comment by hlissner** [comment]
Thanks for your help!
